changelog/2.076.0_pre.dd

```diff
@@ -0,0 +1,374 @@
+$(LI $(BUGZILLA 17721): Wrong expression type using vector extensions with shift operands)
+$(LI $(BUGZILLA 17722): Wrong diagnostic using __traits$(LPAREN)compiles, e1 && e2$(RPAREN) expressions.)
+$(LI $(BUGZILLA 17735): AssertError@ddmd/target.d$(LPAREN)357$(RPAREN): unhandled op using fish operators)
+)
+$(BUGSTITLE DMD Compiler enhancements,
+
+$(LI $(BUGZILLA 9287): DMD should read from stdin when an input file is "-")
+$(LI $(BUGZILLA 9731): Ddoc should output per-paragraph macro instead of $$(LPAREN)DDOC_BLANKLINE$(RPAREN))
+$(LI $(BUGZILLA 17392): Add Dub file for the lexer and parser)
+$(LI $(BUGZILLA 17428): [scope] class field assignment allows to escape scope parameters)
+$(LI $(BUGZILLA 17499): with -betterC switch, call C's assert failure function rather than druntime's)
+$(LI $(BUGZILLA 17521): -betterC programs should not link in Phobos runtime library by default)
+$(LI $(BUGZILLA 17590): Unnecessary GC alloc on returning static local struct)
+$(LI $(BUGZILLA 17697): Ddoc: automatically highlight URLs outside of macro arguments)
+)
+$(BUGSTITLE Phobos regressions,
+
+$(LI $(BUGZILLA 17650): [REG v2.075.0 b1-b4] std.getopt range violation)
+$(LI $(BUGZILLA 17661): New isInputRange rejects valid input range)
+$(LI $(BUGZILLA 17724): digest is not a template declaration, it is a module)
+)
+$(BUGSTITLE Phobos bugs,
+
+$(LI $(BUGZILLA 6718): "nWayUnion" => "nWayMerge", plus true nWayUnion)
+$(LI $(BUGZILLA 12260): Improve error of std.stdio.readf when involving whitespace)
+$(LI $(BUGZILLA 12866): Append to std.container.Array of fixed size arrays)
+$(LI $(BUGZILLA 16079): memoize should cache objects too)
+$(LI $(BUGZILLA 16191): std/digest/digest.d should be renamed to package.d)
+$(LI $(BUGZILLA 17389): std.traits.hasNested fails for class with member of same type)
+$(LI $(BUGZILLA 17458): [std.regex] Assertion failure in package.d)
+$(LI $(BUGZILLA 17539): std.parallellism.parallel triggers 'statement not reachable' warnings in the rdmd_test suite)
+$(LI $(BUGZILLA 17562): Tangent function returns NaN for abs$(LPAREN)input$(RPAREN) >= 2^63)
+$(LI $(BUGZILLA 17574): Range violation in std.getopt:getopt AA parsing)
+$(LI $(BUGZILLA 17616): makeIndex cannot fully use range of index type)
+$(LI $(BUGZILLA 17640): std.concurrenct writeln conflicts with std.stdio writeln in unittests)
+$(LI $(BUGZILLA 17711): std.array.byPair should be usable with const AA)
+)
+$(BUGSTITLE Phobos enhancements,
+
+$(LI $(BUGZILLA 2137): Data not compressed on fly when adding to zip archive)
+$(LI $(BUGZILLA 6384): std.traits.isComparable)
+$(LI $(BUGZILLA 15759): chunks should work with only an input range)
+$(LI $(BUGZILLA 15771): FileLogger should create the output directory if it does not exist)
+$(LI $(BUGZILLA 16062): Add 'clear' method to OutBuffer $(LPAREN)std.outbuffer$(RPAREN))
+$(LI $(BUGZILLA 16744): We should have a TypeOf template so that typeof can be used with templates like staticMap)
```

Comment:
The addition of `TypeOf` was reverted: https://github.com/dlang/phobos/pull/5664
(Just reopened the bug.)